enterprise/server/usage/usage.go

```diff
@@ -0,0 +1,151 @@
+package usage
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/tables"
+	"github.com/buildbuddy-io/buildbuddy/server/util/perms"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"github.com/buildbuddy-io/buildbuddy/server/util/timeutil"
+	"github.com/go-redis/redis"
+	"github.com/jonboulle/clockwork"
```

Comment:
We should be really careful about introducing new 3p packages -- we already depend on too many. Each one increases our security and update burden, so we should really only use stuff we *need*.

Can you use a timer here instead?